server.js

```diff
@@ -87,11 +94,15 @@ if (useCookieSessionStore === 'true') {
 // Authentication middleware must be loaded before other middleware such as
 // static assets to prevent unauthorised access
 middleware.forEach(func => app.use(func))
+app.get('/', (req, res) => {
+  res.send('GOV.UK Prototype Kit (temporary home page)')
+})
```

Comment:
`app/views/index.html` still exists in the prototype starter files, should we delete it?

Reply:
I think we should hold off deleting it until the user research is completed.  This blank homepage is only really there for the integration tests, once we have a direction for the homepage this can be updated.